source/ed/connectors/gtfs_parser.h

```diff
@@ -340,7 +340,7 @@ struct TripsGtfsHandler : public GenericHandler {
 };
 struct StopTimeGtfsHandler : public GenericHandler {
     StopTimeGtfsHandler(GtfsData& gdata, CsvReader& reader) : GenericHandler(gdata, reader) {}
-    int id_c, arrival_c,
+    int trip_c, arrival_c,
```

Comment:
Why not just using the id_c of the mother?

Reply:
It crashed when I did, I can look into it more thoroughly if you think it might be worth it.
Also the naming was not so great here...